File: elovate_frontend/src/dtos/Lobby/lobbyDto.ts
/*-
 * #%L
 * ELOvate
 * %%
 * Copyright (C) 2024 ELOvate GmbH.
 * %%
 * Copyright (C) 2024 ELOvate GmbH. - All Rights Reserved
 * Unauthorized copying of this file, via any medium is strictly prohibited
 * Proprietary and confidential
 * #L%
 */
import {GameDto} from "../Game/gameDto";
import {UserDto} from "../userDto";

export interface LobbyDto {
  id: string;
  game: GameDto,
  teamA: UserDto[]
  teamB: UserDto[]
}

File: elovate_backend/src/main/java/com/tuwien/elovate/dtos/lobby/LobbyDto.java
package com.tuwien.elovate.dtos.lobby;

/*-
 * #%L
 * ELOvate
 * %%
 * Copyright (C) 2024 ELOvate GmbH.
 * %%
 * Copyright (C) 2024 ELOvate GmbH. - All Rights Reserved
 * Unauthorized copying of this file, via any medium is strictly prohibited
 * Proprietary and confidential
 * #L%
 */

import com.tuwien.elovate.dtos.game.GameDto;
import com.tuwien.elovate.dtos.user.UserDto;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class LobbyDto {
    private String id;
    private GameDto game;
    private List<UserDto> teamA;
    private List<UserDto> teamB;
}
